api/src/main/java/com/epam/pipeline/manager/cluster/performancemonitoring/ResourceMonitoringManager.java

```diff
@@ -489,7 +491,9 @@ private void processHighNetworkConsumingRuns(Map<String, PipelineRun> running,
                     if (bandwidth >= bandwidthLimit) {
                         processHighNetworkConsumingRun(run, actionTimeout, action, runsToNotify,
                                 runsToUpdateNotificationTime, bandwidth, runsToUpdateTags);
-                    } else if (run.getLastNetworkConsumptionNotificationTime() != null) {
+                    } else if (run.getLastNetworkConsumptionNotificationTime() != null &&
+                            !run.getTags().containsKey(getNetworkLimitDateTag()) &&
```

Comment:
do we even need to change this code? it seems that logic for limiting and defining if run under network pressure should be separate